### PR TITLE
feat(presets): Add monorepo Spectre.Console

### PR DESCRIPTION
**Pull request by nils-a**
## Changes

Added monorepo preset for [Spectre.Console](https://github.com/spectreconsole/spectre.console)

## Context

If multiple parts of Spectre.Console are consumed in a project, they should all be updated together.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by rarkins** [comment]
The library goes with the name `Spectre.Console` although I think we prefer to avoid dots in our group names? If so then we need to decide between `SpectreConsole` and `Spectre-Console`

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -227,6 +227,7 @@ const repoGroups = {
   'shopify-app-bridge': 'https://github.com/Shopify/app-bridge',
   'sitecore-jss': 'https://github.com/Sitecore/jss',
   skiasharp: 'https://github.com/mono/SkiaSharp',
+  SpectreConsole: 'https://github.com/spectreconsole/spectre.console',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:230]:
```suggestion
  'spectre-console': 'https://github.com/spectreconsole/spectre.console',
```

**Comment by nils-a** [comment]
@rarkins, using `Spectre.Console` also resulted it a bunch of linting errors about "namespaces" - that was the first thing I tried 😀 

When creating the grouping manually, there's the `groupName` and the `groupSluge`, but I found nothing for the built-in groups to provide similar functionality.

In my local setups I currently use:
```json
{
  "matchPackageNames": [
    "Spectre.Console",
    "Spectre.Console.Cli",
    "Spectre.Console.Testing",
    "Spectre.Console.ImageSharp",
    "Spectre.Console.Analyzer"
  ],
  "groupName": "Spectre.Console",
  "groupSlug": "spectre-console"
}
```

I'd rather prefer using `SpectreConsole` over `Spectre-Console`. On the other hand, I have also not found any other grouping here that uses capital letters. So, IMO, it comes down to either `SpectreConsole` which is - within the current frame - closest to the original naming of the project or `spectre-console` which would fit best in with all the other names currently in the file.
As I said, I'd love to see `SpectreConsole`, but I'll also go with whatever fits this project best.

**Comment by viceice** [comment]
it's our design decision to only use lower case and dashes for those groups.

**Comment by nils-a** [comment]
> it's our design decision to only use lower case and dashes for those groups.

good to know. 
I have accepted your suggested edit.

**Review by rarkins — APPROVED** [review]

**Pull request merged**